### ISSUE TITLE
No unresolved reference inspections

### ISSUE THREAD

**Issue by FelixBrakel** [intellij-rust/intellij-rust#4846]
## Environment

* **IntelliJ Rust plugin version: 0.2.113.2150-193**
* **Rust toolchain version: 1.38**
* **IDE name and version: CLion 2019.3.2**
* **Operating system: Arch Linux, kernel 5.4.3**

## Problem description
The unresolved reference inspection never seems to be triggered. I've tested use statements for items that don't exist, function calls to functions that don't exist and referencing structs in mods that don't exist. The inspections never seem to detect the code as faulty. Note that I'm not entirely sure that the 'unresolved reference' inspection is the one causing this but I'm not sure which other one it would be

## Steps to reproduce
Consider this program
```rust
fn main() {
    not_a_func_in_this_scope();
}
```
The function `not_a_func_in_this_scope()` is not detected as being out of the scope by the inspections.

Similarly consider:
```rust
mod data

fn main() {
    data::element::new()
}
```
Where data contains a bunch of structs. The inspections do not detect if the the struct `element` is actually in the data mod.

Or even when there is no mod specified:
```rust
fn main() {
    just::a_random::thing();
}
```

Use statements seems to be in a similar state. literally any use statement can be made, the code inspections won't detect an error as long as it is the correct syntax.
